go.mod

```diff
@@ -1,7 +1,18 @@
 module github.com/blocky/basm-go-sdk
 
-go 1.22.6
+go 1.23.8
```

Comment:
This bump from 1.22 to 1.23 is required to use the testscript library. We may want to split this version upgrade into a different PR ahead of this one and let that propagate to everywhere the SDK is used.

Reply:
not required when not using the testscript library itself. Removing.